### PR TITLE
feat: another parameters of weather

### PR DESCRIPTION
**Pull request by ra1n2006**

### CODE REVIEW

**Comment by yarqr** [comment]
assets/icons/free-icon-storm-3026395.png
assets/icons/free-icon-cloud-4814293.png
...
pls rename this icons such as cloud.png, etc.

UPD: fixed by @DaniilKK2006

File: frontend/result/script.js [hunk 1]
```diff
@@ -4,19 +4,55 @@ let returnButton = document.querySelector(".main__return-button");
...
+        return "/assets/icons/free-icon-fog-3750506.png"
+      case 'Haze':
+        return "/assets/icons/free-icon-fog-3750506.png"
+    }
+  }
+  generImg() {
+    if (this.country !== "UA") {
+      return `https://flagsapi.com/${this.country}/flat/64.png`
+    }
+    return `https://flagsapi.com/RU/flat/64.png`
```
Comment (yarqr) [inline frontend/result/script.js:41]: pls refactor this(more logical then now)

now:
```
if not some_specific_situation:
    return all
return some_specific_result
```
will be
```
if some_specific_situation:
    return some_specific_result
return all
```

File: frontend/result/script.js [hunk 2]
```diff
@@ -4,19 +4,55 @@ let returnButton = document.querySelector(".main__return-button");
...
+}
 async function req() {
   let txt = ``
   let response = await fetch(url);
   let result = await response.json();
   returnButton.removeAttribute("hidden");
   if (result !== null) {
-    txt = `<section> ${city.toUpperCase()}</section> 
+    let weather = new Weather(result.temperature, result.description, result.feels_like, result.country, result.humidity, result.wind_speed)
+    txt = `<section class = "main__res">${city.toUpperCase() + ' '.repeat(150)}<img src=${weather.generImg()} class = "main__res__flag"/></section>
```
Comment (yarqr) [inline frontend/result/script.js]: `' '.repeat(150)` looks like crutch

File: frontend/result/script.js (hunk 1 above)
Comment (yarqr) [inline frontend/result/script.js]: pls refactor this(more logical then now)

now:
```
if not some_specific_situation:
    return all
return some_specific_result
```
will be
```
if some_specific_situation:
    return some_specific_result
return all
```

File: frontend/result/script.js [hunk 3]
```diff
@@ -4,19 +4,55 @@ let returnButton = document.querySelector(".main__return-button");
...
+class Weather {
+  constructor(temperature, description, feels_like, country, humidity, wind_speed) {
+    this.temperature = temperature
+    this.descriprtion = description
+    this.feels_like = feels_like
+    this.country = country
+    this.humidity = humidity
+    this.wind_speed = wind_speed
+  }
+  match() {
```
Comment (yarqr) [inline frontend/result/script.js]: make the name of method more informative

**Comment by yarqr** [comment]
good improve 

> assets/icons/free-icon-storm-3026395.png assets/icons/free-icon-cloud-4814293.png ...
> 
> pls rename this icons such as cloud.png, etc.

**Comment by yarqr** [comment]
missclick

File: frontend/result/script.js (hunk 1 above)
Comment (ra1n2006) [reply frontend/result/script.js]: done

File: frontend/result/script.js (hunk 2 above)
Comment (ra1n2006) [reply frontend/result/script.js]: done

File: frontend/result/script.js (hunk 3 above)
Comment (ra1n2006) [reply frontend/result/script.js]: done

**Pull request merged**